config/config.go

```diff
@@ -574,24 +574,29 @@ func (c *Config) ToRouteSrvOptions() routesrv.Options {
 
 	return routesrv.Options{
 		Address:                            c.Address,
-		SourcePollTimeout:                  time.Duration(c.SourcePollTimeout) * time.Millisecond,
-		WaitForHealthcheckInterval:         c.WaitForHealthcheckInterval,
+		DefaultFiltersDir:                  c.DefaultFiltersDir,
```

Comment:
It's good to see this brought to spotlight: the default filters are a generic functionality, but currently it is only available in Kubernetes. We should fix this separately at some point.